.changeset/friendly-jobs-perform.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+[breaking] disallow unknown exports from `+(layout|page)(.server)?.js` and `+server.js` files
```

Comment:
Do we now want to call out the `_` exception in the changeset?

Either way, now that there's this exception, it should definitely be documented somewhere, and perhaps we could also make the error message nicer. ('If you're exposing this to facilitate testing, consider doing...')